app/questionnaire/questionnaire_schema.py

```diff
@@ -1470,3 +1491,15 @@ def get_calculated_summary_ids_for_grand_calculated_summary(
 
 def is_list_collector_block_editable(block: Mapping) -> bool:
     return bool(block["type"] == "ListCollector")
+
+
+def get_placeholder_answer_ids_for_transform(block: ImmutableDict) -> set[str]:
```

Comment:
`for_transform` implies you are passing it a transform object?. What about `get_placeholder_answer_ids_requiring_routing_path`

Reply:
Updated 👍